Dockerfile

```diff
@@ -46,13 +41,17 @@ RUN apt install curl -y
 COPY options.json /h5ai/_h5ai/private/conf/options.json
 COPY types.json /h5ai/_h5ai/private/conf/types.json 
 
-CMD ["bash", "/root/entrypoint.sh"] 
-
-
 RUN apt install -y php-mbstring
 
 COPY formatAddress.php /h5ai/_h5ai/public/formatAddress.php
 COPY Kekkak.php /h5ai/_h5ai/public/Kekkak.php
 
+# Create nginx configuration
+COPY h5ai-nginx.conf.template /etc/nginx/conf.d/h5ai-nginx.conf.template
```

Comment:
I moved some of this docker commands after `RUN apt install -y php-mbstring`, so that it doesn't have to rerun apt install every time